### PR TITLE
decribe testing and publishing in the docs

### LINKED ISSUE #4: Describe how to add new versions of containers
- mention adding to build and push
- mention adjusting container-tests and openshift tests

### PR DESCRIPTION
**Pull request by zmiklank**
resolves: https://github.com/sclorg/welcome/issues/4

### CODE REVIEW

File: contribution.md [hunk 1]
```diff
@@ -59,9 +66,13 @@ These files mean that the new version will not be built accidentally.
 
 Delete the corresponding `.exclude-$OS` files once the image can built using testing/production rpms.
 
+#### 3) enable publishing
+
+In order to have images available on the [quay.io](quay.io) the `.github/workflows/build-and-push.yml` configuration file need to be updated with the proper $OS and $VERSION.
```
Comment (phracek) [inline contribution.md]:
```suggestion
In order to have images available on the [quay.io](quay.io) the `.github/workflows/build-and-push.yml` configuration file needs to be updated with the proper $OS and $VERSION.
```

Comment (phracek) [inline contribution.md]: I would also add a link to quay.io repositories, where the containers are stored.
E.g. For quay.io/fedora leaves Fedora images, in quay.io/centos7 leaves CentOS 7 images and in quay.io/sclorg goes C8S, C9S images

File: contribution.md [hunk 2]
```diff
@@ -77,8 +88,8 @@ For RHEL, we install packages either from RHEL 7 base or from Red Hat [Software
 
 ## Testing
 
-The images in this organization include test-suites, that test the images in basic use cases. Every pull-request is then tested by running this test-suite. The tests run in ci.centos.org:
-
- * https://ci.centos.org/view/SCLo-images/
+The images in this organization include test-suites, that test the images in basic use cases.
+The tests are being run on [Testing Farm](https://docs.testing-farm.io/general/0.1/index.html). The tests are triggered on demand, when string `[test-all]` for running all tests or `[test]` for running only container tests, is entered in the Pull Request comment. Only maintainers and owners of the SCLOrg can trigger the tests.
```
Comment (phracek) [inline contribution.md]: What about mentioning [test-openshift]?

File: contribution.md [hunk 3]
```diff
@@ -77,8 +88,8 @@ For RHEL, we install packages either from RHEL 7 base or from Red Hat [Software
...
+The person triggering the tests has to check if no malitious code is in the Pull Request.
```
Comment (phracek) [inline contribution.md]: This should be somehow highlighted. 
Also it would be nice to mention, only MEMBER or OWNER can trigger the tests.

**Review by phracek — CHANGES_REQUESTED** [review]

**Comment by phracek** [comment]
@zmiklank Thanks for the PR and review welcome.md file.

File: contribution.md (hunk 2 above)
Comment (zmiklank) [reply contribution.md]: I totally forgot about it :). Will add.

File: contribution.md (hunk 1 above)
Comment (zmiklank) [reply contribution.md]: Sure.

File: contribution.md (hunk 3 above)
Comment (zmiklank) [reply contribution.md]: Yes, that is mentioned there. But I will highlight it.

**Review by phracek — APPROVED** [review]
Thanks for the PR! LGTM

**Pull request merged**